index.html

```diff
@@ -41,442 +19,960 @@
-            />
-            <span class="nav-logo-text"> Turung </span>
-          </a>
-        </div>
-        <div class="nav-btn-con">
-          <a href="#" class="nav-btn-1 d-none">buy tpr</a>
-          <a
-            href="/assets/files/turung_whitepaper.pdf"
-            target="_blank"
-            class="nav-btn-2"
-          >
-            whitepaper
-          </a>
-        </div>
-      </nav>
-      <section class="hero-con">
-        <div class="hero">
-          <!-- <div class="hero-img-con">
-        
-          </div> -->
+
+  <body id="body">
+    <header id="header">
+      <div class="container">
+        <div id="logo" class="pull-left d-flex">
           <img
-            class="hero-img"
-            src="./images/turung-logo.jpg"
+            src="./assets/favicon/favicon-32x32.png"
+            width="100%"
+            height="100%"
             alt="turung logo"
+            class=""
           />
-          <div class="hero-descrip">
-            <h1>turung</h1>
-            <h2>The future of e-commerce is now</h2>
-            <p>...imagine it's possible!</p>
-          </div>
+          <h1>
+            <a href="#body" class="scrollto">Turung</a>
+          </h1>
+          <!-- Uncomment below if you prefer to use an image logo -->
+          <!-- <a href="#body"><img src="img/logo.png" alt="" title="" /></a> -->
         </div>
-        <p>
-          Online is already becoming the most preferred place to feature goods
-          and services and experts had estimated that by the year 2040, 95% of
-          purchases will be through e-commerce platforms.
-          <span class="more" id="more-hero"
-            >Currently, traditional brick and mortar stores have more off-line
-            presence in rural and urban areas compared with their online
-            presence. Turung is designed to provide free online stores to enable
-            off-line stores to sell their products or services beyond their
-            physical location and local currency.
-            </span>
-          <span class="more-btn" id="more-btn-hero">read more</span>
-
-        </p>
-      </section>
-      <article class="article">
-        <div class="article-content_one">
-          <h1>tu<span>rung</span></h1>
-          <div>
-            <p>
-              Turung is an online smartstore that allows traditional brick and
-              mortar businesses to process crypto-fiat liquidity through a
-              decentralized payment system called TURUpay.
-              <span class="more" id="more-articleone">
-                With this payment system, imagine cashing out your Shiba Inu
-                straight into fiat (either as cash or through your bank account)
-                at a local store with one-time transaction fees. This will
-                revolutionize how brick and mortar stores operate by giving
-                access to the $3trillion Crypto market.</span
+
+        <nav id="nav-menu-container">
```

Comment:
![image](https://user-images.githubusercontent.com/60205816/179191557-7456ed68-385b-4df6-9f80-b9518a22655e.png)

I think the colour of the close button on the mobile nav is kinda hard to detect because of the colour, if you can change the colour to a lighter grey or any colour you think is best, that would be great.

Reply:
![image](https://user-images.githubusercontent.com/60205816/179206903-29118c8e-cf4e-4ada-b3f8-6ea2014bf2c8.png)
###  Feedback on Update!
It's looking great. Well done :clap: